integration/query_projection_compat_test.go

```diff
@@ -24,17 +24,68 @@ func TestQueryProjectionCompat(t *testing.T) {
 	t.Parallel()
 
 	testCases := map[string]queryCompatTestCase{
-		"FindProjectionInclusions": {
-			filter:         bson.D{{"_id", "document-composite"}},
-			projection:     bson.D{{"foo", int32(1)}, {"42", true}},
-			skipForTigris:  "Tigris does not support field names started from numbers (`42`)",
-			resultPushdown: true,
-		},
-		"FindProjectionExclusions": {
-			filter:         bson.D{{"_id", "document-composite"}},
-			projection:     bson.D{{"foo", int32(0)}, {"array", false}},
-			skipForTigris:  "Tigris does not support language keyword 'array' as field name",
-			resultPushdown: true,
+		"Include1Field": {
+			filter:     bson.D{},
+			projection: bson.D{{"v", int32(1)}},
+		},
+		"Exclude1Field": {
+			filter:     bson.D{},
+			projection: bson.D{{"v", int32(0)}},
+		},
+		"Include2Fields": {
+			filter:     bson.D{},
+			projection: bson.D{{"foo", int32(1)}, {"bar", true}},
+		},
+		"Exclude2Fields": {
+			filter:     bson.D{},
+			projection: bson.D{{"foo", int32(0)}, {"bar", false}},
+		},
+		"IncludeID": {
+			filter:     bson.D{},
+			projection: bson.D{{"_id", true}},
+		},
+		"ExcludeID": {
+			filter:     bson.D{},
+			projection: bson.D{{"_id", false}},
+		},
+		"IncludeFieldExcludeID": {
+			filter:     bson.D{},
+			projection: bson.D{{"_id", false}, {"array", true}},
```

Comment:
`array` top level field does not exist, is  this testing the case where field does not exist?

Reply:
Changed it to 'v' field.